app/assets/stylesheets/views/_landing_page/hero.scss

```diff
@@ -33,4 +42,16 @@
     padding: govuk-spacing(4);
     margin-top: -(govuk-spacing(6));
   }
+
+  &.app-b-hero__textbox--mid-page {
+    margin: 0;
+
+    @include govuk-media-query($from: tablet) {
+      color: govuk-colour("white");
+      position: absolute;
+      max-width: 336px;
```

Comment:
This seems like a bit of a magic number - I'm worried about the box overlapping the image close to the breakpoints (see below). Could we somehow stick with the govuk-grid widths?

![Screenshot 2024-10-15 at 16 41 44](https://github.com/user-attachments/assets/18a3f936-169a-4136-8128-1438b285072e)

Reply:
Thanks @andysellick for all the help in finding a solution here. This approach involves absolutely positioning the hero image and introducing a new text box wrapper element that acts as an overlay over the hero image. The grid classes can then be applied inside the wrapper without needing custom widths.